### PR TITLE
Add net.biniou.LeBiniou

### PR DESCRIPTION
**Pull request by oliv3**
### Please confirm your submission meets all the criteria

- [x] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [x] My pull request follows the instructions at [App Submission][submission].
- [x] See the manifest for comments on permissions.
- [x] All assets referenced in the manifest are redistributable by any party. 
- [x] I am the author of the project.
- [x] I own the domain used in the application ID.
- [x] **N/A** Any additional patches or files have been submitted to the upstream projects concerned.

[reqs]: https://github.com/flathub/flathub/wiki/App-Requirements
[maint]: https://github.com/flathub/flathub/wiki/App-Maintenance
[submission]: https://github.com/flathub/flathub/wiki/App-Submission

### CODE REVIEW

**Comment by LeandroStanger** [comment]
bot, build net.biniou.LeBiniou

File: net.biniou.LeBiniou.yml [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+      - make -j4 install
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --socket=pulseaudio
+  - --filesystem=xdg-run/pipewire-0
+  # used to send anonymous usage statistics
+  - --share=network
+  # users can put configuration and/or data files anywhere
+  - --filesystem=home
```
Comment (barthalion) [inline net.biniou.LeBiniou.yml:99]: Ultimately it's your decision, but I'd limit it to xdg-music and xdg-download. Users can change these permissions locally.

File: net.biniou.LeBiniou.yml [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+         url: https://gitlab.com/lebiniou/lebiniou.git
+         tag: version-3.52.0
+         commit: 02fc30edad6ef20396a8400104edb5bea539e308
+    build-commands:
+      - autoreconf -i
+      - ./configure --prefix=/app
+      - make -j4 install
+finish-args:
+  - --share=ipc
+  - --socket=x11
```
Comment (barthalion) [inline net.biniou.LeBiniou.yml:93]: Does it support Wayland?

File: net.biniou.LeBiniou.yml [hunk 3]
```diff
@@ -0,0 +1,101 @@
+app-id: net.biniou.LeBiniou
+runtime: org.freedesktop.Platform
+runtime-version: '20.08'
+sdk: org.freedesktop.Sdk
+command: lebiniou
+modules:
+  - name: microhttpd
+    buildsystem: simple
```
[the remark this answers is not in the document]
Comment (oliv3) [reply net.biniou.LeBiniou.yml]: Fixed.

File: net.biniou.LeBiniou.yml [hunk 4]
```diff
@@ -0,0 +1,101 @@
...
+    sources:
+       - type: git
+         url: https://gitlab.com/lebiniou/lebiniou.git
+         tag: version-3.52.0
+         commit: 02fc30edad6ef20396a8400104edb5bea539e308
+    build-commands:
+      - autoreconf -i
+      - ./configure --prefix=/app
+      - make -j4 install
+finish-args:
```
[the remark this answers is not in the document]
Comment (oliv3) [reply net.biniou.LeBiniou.yml]: Fixed.

File: net.biniou.LeBiniou.yml (hunk 1 above)
Comment (oliv3) [reply net.biniou.LeBiniou.yml]: I'll leave it as is for now.

File: net.biniou.LeBiniou.yml (hunk 2 above)
Comment (oliv3) [reply net.biniou.LeBiniou.yml]: I don't see why it wouldn't ?

Comment (oliv3) [reply net.biniou.LeBiniou.yml]: Fixed.

**Comment by nedrichards** [comment]
bot, build net.biniou.LeBiniou

File: net.biniou.LeBiniou.yml [hunk 5]
```diff
@@ -0,0 +1,86 @@
+app-id: net.biniou.LeBiniou
+runtime: org.freedesktop.Platform
+runtime-version: '20.08'
+sdk: org.freedesktop.Sdk
+command: lebiniou
+finish-args:
+  - --share=ipc
+  - --socket=x11
```
Comment (SISheogorath) [inline net.biniou.LeBiniou.yml:8]: When it supports wayland, this should become `fallback-x11`

**Comment by oliv3** [comment]
Ok.

Le 4 janvier 2021 22:35:19 GMT+01:00, Sheogorath <notifications@github.com> a écrit :
>@SISheogorath commented on this pull request.
>
>
>
>> @@ -0,0 +1,86 @@
>+app-id: net.biniou.LeBiniou
>+runtime: org.freedesktop.Platform
>+runtime-version: '20.08'
>+sdk: org.freedesktop.Sdk
>+command: lebiniou
>+finish-args:
>+  - --share=ipc
>+  - --socket=x11
>
>When it supports wayland, this should become `fallback-x11`
>
>-- 
>You are receiving this because you authored the thread.
>Reply to this email directly or view it on GitHub:
>https://github.com/flathub/flathub/pull/2023#pullrequestreview-561361077

-- 
Envoyé de mon appareil Android avec Courriel K-9 Mail. Veuillez excuser ma brièveté.

File: net.biniou.LeBiniou.yml (hunk 5 above)
Comment (oliv3) [reply net.biniou.LeBiniou.yml:8]: Ok, thanks.

File: net.biniou.LeBiniou.yml [hunk 6]
```diff
@@ -0,0 +1,86 @@
...
+    buildsystem: simple
+    sources:
+       - type: git
+         url: https://gitlab.com/lebiniou/lebiniou.git
+         tag: version-3.52.0
+         commit: 02fc30edad6ef20396a8400104edb5bea539e308
+    build-commands:
+      - autoreconf -i
+      - ./configure --prefix=/app
+      - make -j4 install
```
Comment (nedrichards) [inline net.biniou.LeBiniou.yml:86]: This should be $FLATPAK_BUILDER_N_JOBS like so: https://github.com/linuxhw/hw-probe/blob/07ff9f20b5ebc23db596691037d04e0eef460f3b/flatpak/org.linux_hardware.hw-probe.yaml#L81

File: net.biniou.LeBiniou.yml [hunk 7]
```diff
@@ -0,0 +1,86 @@
...
+    buildsystem: simple
+    sources:
+       - type: git
+         url: https://gitlab.com/lebiniou/lebiniou-data.git
+         tag: version-3.50
+         commit: f4b03c90b838c4b84ab2cd56348dedf724e653b2
+    build-commands:
+      - autoreconf -i
+      - ./configure --prefix=/app
+      - make -j4 install
```
Comment (nedrichards) [inline net.biniou.LeBiniou.yml:75]: This should be $FLATPAK_BUILDER_N_JOBS like so: https://github.com/linuxhw/hw-probe/blob/07ff9f20b5ebc23db596691037d04e0eef460f3b/flatpak/org.linux_hardware.hw-probe.yaml#L81

**Comment by nedrichards** [comment]
lgtm in testing, just a response on the make -j question and it's good to merge for me.

File: net.biniou.LeBiniou.yml (hunk 6 above)
Comment (oliv3) [reply net.biniou.LeBiniou.yml:86]: @nedrichards Thanks, I'll fix this soon as well as the `fallback-x11` issue, please stay tuned.

**Comment by oliv3** [comment]
@nedrichards All issues should be addressed now.

**Comment by nedrichards** [comment]
bot, build net.biniou.LeBiniou

**Pull request closed without merging**